backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -27,8 +37,14 @@
  */
 public class BacktraceBase implements Client {
 
+    static {
+        System.loadLibrary("backtrace-native");
+    }
+
     private static transient String LOG_TAG = BacktraceBase.class.getSimpleName();
 
+
+    public native void Crash();
```

Comment:
in Java it is normal to start the name of a method from a lower case letter

Reply:
done